Tests/Packet++Test/Tests/SipSdpTests.cpp

```diff
@@ -648,6 +648,22 @@ PTF_TEST_CASE(SdpLayerParsingTest)
 } // SdpLayerParsingTest
 
 
+PTF_TEST_CASE(NotSdpLayerParseingTest)
+{
+	timeval time;
```

Comment:
we don't need this?

Reply:
please remove unused code.

Reply:
I see that the macro `READ_FILE_AND_CREATE_PACKET` need the argument `time`, so this argument needs to be retained.

Reply:
yeah... it's a weird design ...